### PR TITLE
Features/button outline

### PR DESCRIPTION
**Pull request by yinchyy**
Add outline style to button component

### CODE REVIEW

File: src/config/backgrounds.scss [hunk 1]
```diff
@@ -17,39 +17,89 @@
...
+    color: $color-disabled !important;
+    &:hover {
+      background-color: white !important;
+    }
+  }
+}
 
 .backgroundPrimary {
   @include background($color-primary, white);
+  &.outlined {
```
Comment (pilotpirxie) [inline src/config/backgrounds.scss:46]: try with if

File: src/config/backgrounds.scss [hunk 2]
```diff
@@ -17,39 +17,89 @@
     color: lighten($color, 40%);
   }
 }
+@mixin outline($bgColor) {
+  background-color: white !important;
```
Comment (pilotpirxie) [inline src/config/backgrounds.scss:21]: transparent

File: src/config/backgrounds.scss [hunk 3]
```diff
@@ -17,39 +17,89 @@
...
+  color: $bgColor !important;
```
Comment (pilotpirxie) [inline src/config/backgrounds.scss:22]: try to get rid of importants if you can (optional)

Comment (yinchyy) [reply src/config/backgrounds.scss]: resolved in [ca69a39](https://github.com/pilotpirxie/react-toucan-ui/pull/2/commits/ca69a3905bcb49f4027db7dbd21a3ee72f2f0b2d)

File: src/config/backgrounds.scss (hunk 1 above)
Comment (yinchyy) [reply src/config/backgrounds.scss:47]: resolved in [ca69a39](https://github.com/pilotpirxie/react-toucan-ui/pull/2/commits/ca69a3905bcb49f4027db7dbd21a3ee72f2f0b2d)

File: src/config/backgrounds.scss [hunk 4]
```diff
@@ -1,55 +1,106 @@
...
 .backgroundTertiary {
   @include background($color-tertiary, black);
+  &.outlined {
+    @include background($color-tertiary, white, "outline");
+  }
 }
 
 .backgroundSuccess {
   @include background($color-success, white);
+  &.outlined {
```
Comment (pilotpirxie) [inline src/config/backgrounds.scss:68]: maybe you can simplify here with
:not()
https://developer.mozilla.org/en-US/docs/Web/CSS/:not

so &.outlined {}
&:not(.outlined){}

idk if that works, can you try?

File: src/Button/Button.tsx [hunk 5]
```diff
@@ -25,30 +38,33 @@ export type ButtonProps = {
...
-        [styles.backgroundInfo]: color === 'info',
-        [styles.backgroundWarning]: color === 'warning',
-        [styles.backgroundDanger]: color === 'danger',
-        [styles.backgroundLight]: color === 'light',
-        [styles.backgroundDark]: color === 'dark',
+        [styles.backgroundPrimary]: color === "primary",
+        [styles.backgroundSecondary]: color === "secondary",
+        [styles.backgroundTertiary]: color === "tertiary",
+        [styles.backgroundSuccess]: color === "success",
+        [styles.backgroundInfo]: color === "info",
```
Comment (pilotpirxie) [inline src/Button/Button.tsx:55]: nothing changed in that line besides " -> ' check if linter is consistent

**Review by pilotpirxie — APPROVED** [review]

**Pull request merged**